plugins/grpc-ecosystem/gateway/v2.21.0/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# syntax=docker/dockerfile:1.7
```

Comment:
```suggestion
# syntax=docker/dockerfile:1.9
```

Reply:
It would be nice to automate this at some point, should be a quick post process function that reads from 

https://github.com/bufbuild/plugins/blob/b7164d301205734fdf4d8a9d7f452c7be57a53e8/.github/docker/Dockerfile.dockerfile#L1

And then does a crude search/replace on this, similar to the other automation we have:

https://github.com/bufbuild/plugins/blob/d4947709/internal/cmd/fetcher/main.go#L59-L76